packages/terra-clinical-onset-picker/src/OnsetPicker.scss

```diff
@@ -12,4 +12,26 @@
     margin-bottom: 0.357em;
```

Comment:
Let's make sure and create themeable variables for these values.  Take a look at other components for what/how we typically theme.

Reply:
Do you want just these margins to be made into variables? Or should I also include the min-width values?

Reply:
Made the margins and widths themeable variables. I also converted the widths to `em`'s